vita3k/gui/src/live_area.cpp

```diff
@@ -820,12 +820,12 @@ void draw_live_area_screen(GuiState &gui, EmuEnvState &emuenv) {
                 std::vector<ImVec4> str_color;
 
                 if (!str_tag.color.empty()) {
-                    unsigned int color = 0xFFFFFFFF;
+                    uint32_t color = std::numeric_limits<uint32_t>::max();
```

Comment:
It's not uint32_t max value, it's ARGB hex color (white non-transparent). I think it's better to let colors be in hex values as they are used in web. Or use some constant (I don't remember if there is a constant for white color)